lib/analyze_hoc_activity_helper.rb

```diff
@@ -30,12 +30,25 @@ def analyze_day_fast(date)
     " WHERE (finished_at >= '#{day}' AND finished_at < '#{next_day}')" +
     "   OR (pixel_finished_at >= '#{day}' AND pixel_finished_at < '#{next_day}')"
 
+  # Generate a list of Code.org tutorials so that we can generate the count for
+  # Code.org hosted tutorials below.
+  codedotorg_tutorials = []
+  PEGASUS_REPORTING_DB_READONLY.fetch(
+    "SELECT code FROM tutorials WHERE orgname = 'Code.org'"
```

Comment:
I don't understand how the query missed these tutorials when they also have org name Code.org?

What if we added a column code_studio_b?